pkg/server/datastore/sqlstore/migration.go

```diff
@@ -403,6 +406,13 @@ func migrateToV20(tx *gorm.DB) error {
 	return nil
 }
 
+func migrateToV21(tx *gorm.DB) error {
+	if err := tx.AutoMigrate(&RegisteredEntry{}, &Selector{}).Error; err != nil {
```

Comment:
there are no changes on Selector struct
```suggestion
	if err := tx.AutoMigrate(&RegisteredEntry{}).Error; err != nil {
```